README.md

```diff
@@ -1 +1,18 @@
-# Frends.Cryptography
\ No newline at end of file
+# Frends.Cryptography
+
+Frends task for generating strings to chosen HashAlgorithm type.
```

Comment:
I don't think this will be true for all tasks that will be in this repository, since there might be some other stuff related to cryptography other than calculating hashes. The repo description should be a bit more generic related to cryptography.